package.json

```diff
@@ -9,6 +9,7 @@
     "build-clean": "npm run clean && npm run build",
     "test-simple": "nyc mocha \"./test/**/*.spec.js\"",
     "test": "npm run clean && npm run gardener && npm run test-simple",
+    "coverage": "npm run build-clean && nyc --reporter=html --reporter=text mocha",
```

Comment:
This shouldn't be necessary. You can just run

`yarn run test-simple --verbose`


and then keep this open in your browser

`chrome coverage/lcov-report/index.html`

Reply:
ok